solr/core/src/test/org/apache/solr/handler/TestSQLHandler.java

```diff
@@ -2388,6 +2388,7 @@ public void testMultiValuedFieldHandling() throws Exception {
     update.add("id", String.valueOf(maxDocs)); // all multi-valued fields are null
     update.commit(cluster.getSolrClient(), COLLECTIONORALIAS);
 
+    expectResults("SELECT stringxmv, stringsx, booleans FROM $ALIAS WHERE stringxmv IN ('a') AND stringxmv IN ('b')", 10);
```

Comment:
how is this working? is calcite just matching all rows here? I thought the bug here was that calcite was erasing the two IN's and then matching none ;-)

Reply:
that was a temporary change that got pushed accidentally. the assert actually fails. I have removed it in the next commit